template.rb

```diff
@@ -120,6 +121,16 @@ def print_error_message
   EOT
 end
 
+def print_default_addons_message
```

Comment:
the method name sounds generic...can we choose another name, many `post_default_addons_install`, maybe

Reply:
also fixed in 281eafb.